api/v1/endpoints/queries.sql

```diff
@@ -65,7 +68,8 @@ WITH object_data AS (
         JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
         JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
         JOIN assessment_instances AS ai ON (ai.assessment_id = a.id)
-        JOIN users AS u ON (u.user_id = ai.user_id)
+        LEFT JOIN group_info(a.id) AS gi ON (gi.id = ai.group_id)
+        LEFT JOIN users AS u ON (u.user_id = ai.user_id)
```

Comment:
This was necessary because `assessment_instance.user_id` is null for group work.

Ultimately, group work will have non-null values for `group_id`, `group_name`, and `group_uid_list` and null values for `user_uid`, `user_name`, and `user_id`. The opposite will be true for non-group work.

The name `group_uid_list` gives us some flexibility in the future if we want to introduce a property like `group_users` that would allow us to include more information to mirror what's available for non-group assessment instances:

```json
{
  "group_users": [
    {
      "user_name": "Foo",
      "user_id": 1234,
      "user_uid": "foo@bar.com"
    }
  ]
}
```

Although now I'm thinking that `group_uids` might be better than `group_uids_list`? We don't use the `_list` suffix anywhere else in the API AFAIK.

Reply:
Is there a possible fix to this bug? I was comparing code blocks between instructurepages and API/v1, not quite familiar with the project structure. Appreciate any effort on fixing this !!! Thanks in advance.